### PR TITLE
Initial tweaking of previous poll period

### PR DESCRIPTION
**Pull request by tyeth**
Will close https://github.com/adafruit/Adafruit_Wippersnapper_Arduino/issues/463

### CODE REVIEW

**Comment by tyeth** [comment]
@brentru tested on S2 TFT with LTR390, period set at 5mins, immediate data sent.

File: src/components/digitalIO/Wippersnapper_DigitalGPIO.cpp [hunk 1]
```diff
@@ -116,12 +116,15 @@ void Wippersnapper_DigitalGPIO::initDigitalPin(
              pinName, periodMs);
     WS._ui_helper->add_text_to_terminal(buffer);
 #endif
+    // get current time
+    long curtime = millis() - 1;
```
Comment (brentru) [inline src/components/digitalIO/Wippersnapper_DigitalGPIO.cpp:120]: `curTime`

File: src/components/i2c/drivers/WipperSnapper_I2C_Driver.h [hunk 2]
```diff
@@ -1314,97 +1314,123 @@ class WipperSnapper_I2C_Driver {
     setSensorProximityPeriod(period);
   }
 
+  long SENSOR_STARTING_PERIOD_PRV =
```
Comment (brentru) [inline src/components/i2c/drivers/WipperSnapper_I2C_Driver.h:1317]: 1) Where did this formula come from?
2) Caps and case - `sensorStartingPeriodPrv`

File: src/components/digitalIO/Wippersnapper_DigitalGPIO.cpp (hunk 1 above)
Comment (brentru) [inline src/components/digitalIO/Wippersnapper_DigitalGPIO.cpp:120]: Do we want this to be a `long`, millis() returns an `unsigned long`

**Review by brentru — CHANGES_REQUESTED** [review]
This addresses a big issue, thank you! Requesting clarification and changes

File: src/components/i2c/drivers/WipperSnapper_I2C_Driver.h (hunk 2 above)
Comment (tyeth) [reply src/components/i2c/drivers/WipperSnapper_I2C_Driver.h:1317]: Thinking yes swap to a macro. The formula was 24hrs in milliseconds, to account for the unknown period (max of 24hrs in wippersnapper)

**Pull request merged**